.circleci/config.yml

```diff
@@ -269,8 +269,8 @@ workflows:
           ruby_version: '3.0'
           ruby_opt: -W:deprecated
       - tests_macos:
-          name: 'Execute tests on macOS (Xcode 13.0.0, Ruby 3.1)'
-          xcode_version: '13.0.0'
+          name: 'Execute tests on macOS (Xcode 13.3.1, Ruby 3.1)'
+          xcode_version: '13.3.1'
```

Comment:
why not `Xcode14` ? 🤔

Reply:
I mistakenly thought I had to test on Xcode13.　Now that I understand that the main purpose is to test on Ruby 3.1, I'll bump it up to Xcode14.